src/libLookup/Lookup.cpp

```diff
@@ -125,7 +125,7 @@ void Lookup::InitSync() {
       LOG_GENERAL(INFO,
                   "TxBlockNum " << txBlockNum << " DSBlockNum: " << dsBlockNum);
       ComposeAndSendGetDirectoryBlocksFromSeed(
-          m_mediator.m_blocklinkchain.GetLatestIndex() + 1);
+          m_mediator.m_blocklinkchain.GetLatestIndex() + 1, LOOKUP_NODE_MODE);
```

Comment:
Note that we have additional param `toSendSeed` which is defaulted to true otherwise. With above change it is being set again instead of last new param `includeminerinfo`.
This should fix the issue - 
```
ComposeAndSendGetDirectoryBlocksFromSeed(m_mediator.m_blocklinkchain.GetLatestIndex() + 1, true, LOOKUP_NODE_MODE);
```